examples/mnit_ballz/src/objects.nit

```diff
@@ -0,0 +1,90 @@
+# Module containing all objects of the game
+module objects
+
+import geometry
+import geometry::polygon
+import geometry::boxes
+import collision
+
+# The ball is the main character of the game !
+class Ball
+	#center of the ball
+	var center: Point[Float] is writable
+end
+
+# Walls make the ball bounce on them
+class Wall
```

Comment:
yes, they're not playable, but it could be fun xD, will note that somewhere thx for the idea